src/test/java/com/pusher/rest/PusherTest.java

```diff
@@ -205,7 +205,8 @@ public void channelList() throws Exception {
     @Test
     public void channelListLimitOverLimit() {
         Assertions.assertThrows(IllegalArgumentException.class, () -> {
-            final List<String> channels = Arrays.asList(new String[] { "one", "two", "three", "four", "five", "six", "seven", "eight", "nine", "ten", "eleven" });
+
+            final List<String> channels = Arrays.asList(new String[101]);
```

Comment:
This test was failing on `master`, i.e. before that PR even... I'm a bit surprised/scared, do we have a CI?

Reply:
I don't see it failing on `master` :thinking:, perhaps it's due to different configurations, I'll have a look.

Reply:
[This commit](https://github.com/pusher/pusher-http-java/commit/ba4347e984e79f49e2ee3c8742036c97cc09ed60) changed the limit, but the [corresponding test](https://github.com/pusher/pusher-http-java/blob/master/src/test/java/com/pusher/rest/PusherTest.java#L206) didn't get updated.

Reply:
Hmm, just ran `mvn clean verify`, looks like tests aren't actually run:

![image](https://user-images.githubusercontent.com/4568204/73769117-08efd280-4772-11ea-820a-06cfcbb3fa30.png)

No too sure how Travis does though, but it runs a similar command automagically IIRC?

Reply:
Found our culprit: https://stackoverflow.com/questions/36970384/surefire-is-not-picking-up-junit-5-tests 😁

Reply:
well.. that's.. not great. I'll check when they stopped running :man_facepalming:

Reply:
the problem was introduced here:
https://github.com/pusher/pusher-http-java/commit/a17ee634cfcb860d62aa45b955d543e38689d28d
I'll try to fix that before merging, should get it done today.